client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/index.tsx

```diff
@@ -9,6 +9,7 @@ import { MigrationStatus } from 'calypso/data/site-migration/landing/types';
 import { useUpdateMigrationStatus } from 'calypso/data/site-migration/landing/use-update-migration-status';
 import { useSiteIdParam } from 'calypso/landing/stepper/hooks/use-site-id-param';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ImporterPlatform } from 'calypso/lib/importer/types';
```

Comment:
Nit: another type import
```suggestion
import type { ImporterPlatform } from 'calypso/lib/importer/types';
```